packages/camera/camera_windows/windows/camera_plugin.cpp

```diff
@@ -341,6 +346,52 @@ void CameraPlugin::StopVideoRecording(
   }
 }
 
+std::optional<FlutterError> CameraPlugin::StartImageStream(int64_t camera_id) {
+  Camera* camera = GetCameraByCameraId(camera_id);
+  if (!camera) {
+    return FlutterError("camera_error", "Camera not created");
+  }
+
+  CaptureController* cc = camera->GetCaptureController();
+  assert(cc);
+
+  std::ostringstream event_channel_name;
+  event_channel_name << "plugins.flutter.io/camera_windows/imageStream/"
+                     << camera_id;
+
+  auto frame_event_channel = std::make_unique<flutter::EventChannel<>>(
+      messenger_, event_channel_name.str(),
+      &flutter::StandardMethodCodec::GetInstance());
+
+  auto event_channel_handler =
+      std::make_unique<flutter::StreamHandlerFunctions<>>(
+          [cc = cc](auto arguments, auto events) {
+            cc->StartImageStream(std::move(events));
+            return nullptr;
+          },
+          [cc = cc](auto arguments) {
+            cc->StopImageStream();
+            return nullptr;
+          });
+
+  frame_event_channel->SetStreamHandler(std::move(event_channel_handler));
```

Comment:
what happens if stream is started, stopped and started again for same camera.
does it create multiple event channels for same camera.
should channel be stored and cleared at StopImageStream method call? with
frame_event_channel->SetStreamHandler(null_ptr) or something?